README.md

```diff
@@ -23,6 +23,7 @@ Now that we've gotten this out of the way here's an example
 const ngBanks = require('ng-banks');
 
 const banks = ngBanks.getBanks();
+const singleBank = ngBanks.getBank('UBA');
```

Comment:
Add a comment before this to let folks know that this is to get a single bank by it's slug.